packages/api/endpoints/granules.js

```diff
@@ -136,10 +136,11 @@ const update = async (req, res) => {
     updatedBody = restOfBody;
     message = ' Skipped setting status to queued because granule was not running';
   }
-
+  const now = Date.now();
   const updatedGranule = {
     ...existingGranule,
-    updatedAt: Date.now(),
+    updatedAt: now,
+    timestamp: now,
```

Comment:
Why was this needed, versus removing the keys, given that in `generateGranuleRecord` we have 

```
      timestamp: new Date(timestamp).valueOf() || now,
      updatedAt: new Date(updatedAt).valueOf() || now,
```?

If we have defaults set in the generation code, perhaps we shouldn't duplicate that in endpoint logic, and have the endpoint remove values it wants to default for <reasons>?   Open to discussion/pushback on that.

Reply:
I did consider just removing the values from the existing granule as they do have default values.
This seemed more obvious to me, but happy to make it however.

Reply:
I'm mostly an un-fan of having object creation defaults in multiple levels when the reason for the default is the same because it makes situations where the behavior needs to change ... difficult.   That said, there's not a good reason to spend too much time fretting over it here, particularly given we're going to be refactoring this to not use the api object model in the near future.